### PR TITLE
Blur offensive image

### PR DESCRIPTION
**Pull request by lmcnulty**
Makes the offensive image blurred and shown on click, as mentioned in #917. Also sets the blog post date to the 11th.

"Sensitive: click to view" still needs to be translated.

### CODE REVIEW

**Comment by cesarvarela** [comment]
A couple of things should be improved:

- add the component as a short code, so there is no need to import it. Example: https://github.com/responsible-ai-collaborative/aiid/blob/staging/site/gatsby-site/src/components/ui/MdxComponents.js#L25
- instead of loading the image statically it should use Gatsby's image pipeline (it should use`gatsby-plugin-image`)

No need to do this right now because we are already using the `Details` component for this, and it is good enough to go live with it, but this is definitively nice to have.

**Comment by lmcnulty** [comment]
@cesarvarela I've made those change and added alt-text to the images on the English blog post while was there.

File: site/gatsby-site/blog/multilingual-incident-reporting/index.mdx [hunk 1]
```diff
@@ -8,18 +8,20 @@ author: 'Sean McGregor and Cesar Varela'
 slug: '/blog/multilingual-incident-reporting'
 ---
 
+import SensitiveImage from '../../src/components/images/sensitive';
```
Comment (cesarvarela) [inline site/gatsby-site/blog/multilingual-incident-reporting/index.mdx:11]: This shouldn't be necessary

**Review by cesarvarela — COMMENTED** [review]
Removing the extra `import` and good to go 👍

**Pull request merged**